docs/reference/src/language/builtins/elements.md

```diff
@@ -656,6 +656,78 @@ export component Example inherits Window {
 }
 ```
 
+## `SwipeGestureRecognizer`
+
+The `SwipeGestureRecognizer` is used to recognize swipes gesture on the screen in some particular direction.
+Press events on it are still forwarded to the children, but with a small delay.
+If the cursor has moved by more than some internal threshold, a swipe gesture is recognized, and events are no
+longer forwared to the children.
+
+### Properties
+
+-   **`enabled`** (_in_ _bool_): When not enabled, the `SwipeGestureRecognizer` is not recognizing any events.
+    (default value: `true`)
+-   **`swipe-left`**, **`swipe-right`**, **`swipe-up`**, **`swipe-down`** (_out_ _bool_): Recognizes a swipe in
+    the corresponding direction. (default value: `false`)
```

Comment:
```suggestion
-   **`swipe-left`**, **`swipe-right`**, **`swipe-up`**, **`swipe-down`** (_out_ _bool_): Enable recognition of  swipes in
    the corresponding direction. (default value: `false`)
```

I'm also torn about the names here. `swipe-left` sounds like an action, while here it means: Please reocgnize left-bound swipe gestures. Should this be called differently perhaps? `recognize-swip-left: true;` ?

Reply:
i've got no attachment to the names. Yes, maybe `recognize-swip-left` is better